config/config.json5

```diff
@@ -0,0 +1,234 @@
+            unbanBitcoinsForBarters: true,
+        },
+        otherFleaMarketChanges: {
+            // Master Toggle for all other flea market changes below.
+            enabled: true,
+            // NOT RECOMMENDED TO CHANGE, use TraderChanges options below. If option is CHANGED below, it will ALLOW selling on flea market (default is NO SELLING ON FLEA). Will allow you to sell weapons and other blocked by Softcore items like in Vanilla. Not playtested and balanced around this, not recommended to enable.
+            sellingOnFlea: false,
+            // PMC level flea market opens at.
+            fleaMarketOpenAtLevel: 5,
+            // Slightly increase flea market prices to stimulate looting and crafting, instead of buying everything on flea. With barter economy and variance enabled you still get a lot of great trades lower then actual item value. Hustle!
+            fleaPricesIncreased: 1.3,
+            // Only pristine condition items are offered on flea.
+            fleaPristineItems: true,
+            // Be a man, don't change this. This is borderline cheating option for infinite money because of variance changes.
+            onlyFoundInRaidItemsAllowedForBarters: true,
+        },
+    },
+    traderChanges: {
+        // Master toggle for all trader changes below
+        enabled: true,
+        // Adjusts trader evaluations for selling items (from Prapor 50%, Therapist 63%, Ragman 62%, Jaeger 60%, Mechanic 56%, Peacekeeper 45%, Skier 49% --> Prapor 60%, Therapist 64%, Ragman 63%, Jaeger 62%, Mechanic 61%, Peacekeeper 58%, Skier 59% at loyalty level 1). Also every loyalty level will give 5% better selling evaluation so e.g. at LL4 Propor will buy items for 75% of their value.
+        betterSalesToTraders: true,
+        // Nerfs Therapist buying categories (instead of all barter items, she buys only meds and household goods, good for trader diversity), allows Ragman to buy valuables and Skier to buy info items.
+        alternativeCategories: true,
+        pacifistFence: {
+            // To go algong with the theme of this mod, Fence also sells only pacifist items. His prices are dependant on scav karma, so at 6 karma he will sell items almost at the same price Therapist buys them from you.
+            enabled: true,
+            numberOfFenceOffers: 30,
+        },
+        // Rebalances costs of hideout cases to fair and resonable prices and barters. Also changes LEDX dogtag barter
+        reasonablyPricedCases: true,
+        // EXPERIMENTAL option, makes Skier use Euros for all trades, and quest rewards, mostly just for fun and diversity. Adjusts assorts and loyalty levels accordingly. Turned off by default, because needs adjusting existing profiles salesSum (you can do it manually, find TradersInfo in your %profileID%.json, and in 58330581ace78e27b8b10cee key divide salesSum by 115 without leaving the remainder), unless you start a profile with this enabled.
+        skierUsesEuros: true,
```

Comment:
The comment indicates this is disabled by default, but it is enabled by default. Leftover from testing, maybe?

Reply:
Yes this is a leftover from testing. Updated, thanks 👍